CHANGES/8197.feature

```diff
@@ -0,0 +1,2 @@
+Added a `namespace_is_username` helper to decide whether the namespace matches the username of the requests user.
```

Comment:
please update the  docs this is mentioned there as well https://docs.pulpproject.org/pulp_container/role-based-access-control.html#namespaces